dev/org.eclipse.codewind.openapi.feature.test/feature.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2019 IBM Corporation and others.
+    This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<feature
+      id="org.eclipse.codewind.openapi.feature.test"
+      label="%featureName"
+      version="1.0.200.qualifier"
```

Comment:
This is the main test feature and the test update site zip will be based on this.  Should we change the version to be "0.7.0.qualifier" ?